src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -336,8 +336,6 @@ private void LoadTests(ICallbackEventHandler handler)
 
         private void ExploreTests(ICallbackEventHandler handler, string filter)
         {
-            Guard.ArgumentNotNull(filter, "filter");
-
```

Comment:
I know that, but if we are handling an empty string, why not handle null? Why force people to pass in an empty string?